edit-post/components/header/fixed-toolbar-toggle/index.js

```diff
@@ -8,23 +8,18 @@ import { withSelect, withDispatch } from '@wordpress/data';
  */
 import { __ } from '@wordpress/i18n';
 import { compose } from '@wordpress/element';
-import { MenuGroup, MenuItem, withInstanceId } from '@wordpress/components';
+import { MenuItem } from '@wordpress/components';
 import { ifViewportMatches } from '@wordpress/viewport';
 
-function FeatureToggle( { onToggle, isActive } ) {
+function FixedToolbarToggle( { onToggle, isActive } ) {
```

Comment:
While an opposing argument may be made that the unrelated changes are a distraction to the review, I appreciate the small refactorings made in passing here between the component name correction and dropping the unused `withInstanceId`. ❤️